src/sentry/testutils/cases.py

```diff
@@ -923,9 +923,17 @@ def init_snuba(self):
     def store_event(self, *args, **kwargs):
         with mock.patch("sentry.eventstream.insert", self.snuba_eventstream.insert):
             stored_event = Factories.store_event(*args, **kwargs)
+
+            # Error groups
             stored_group = stored_event.group
             if stored_group is not None:
                 self.store_group(stored_group)
+
+            # Performance groups
+            stored_groups = stored_event.groups
+            if stored_groups is not None:
+                for group in stored_groups:
+                    self.store_group(group)
```

Comment:
This isn't optimal but we do have two cases of having a `group` or `groups`

Reply:
`group` will eventually go away and be replaced by `groups` so this is fine for now.